quickwit/quickwit-janitor/src/actors/delete_task_pipeline.rs

```diff
@@ -241,15 +241,15 @@ impl DeleteTaskPipeline {
 }
 
 #[derive(Debug)]
-struct SuperviseLoop;
+pub(crate) struct DeletePipelineSuperviseLoop;
```

Comment:
```suggestion
struct DeletePipelineSuperviseLoop;
```

That's an horrible bug.

I assume you are trying to trigger a delete pipeline supervise event?
But everytime you call it externally you are starting a new loop!

Reply:
oh... this one did not cross my mind :/ 
I definitely should be more careful with "pub" "pub(crate)" stuff

Reply:
wtf, why did I code this... I was probably in a parallel universe :/